packages/x-date-pickers/src/AdapterDayjs/index.ts

```diff
@@ -46,25 +46,36 @@ export class AdapterDayjs extends BaseAdapterDayjs implements MuiPickersAdapter<
 
   constructor(options: Opts) {
```

Comment:
See https://github.com/mui/mui-x/issues/6948#issuecomment-1323220551

`dayjs.utc` is almost a full instance of `dayjs`
It just lack the `extend` and `Ls` properties.